### DISCUSSION TITLE
A Windows update messed up my entire Scoop setup

### DISCUSSION THREAD

**Discussion by Miqueas** [ScoopInstaller/Scoop#6185] [Help and Questions] [answered]
So last night Windows got update in my PC, but after it finished, all my Scoop setup was completely messed up, the apps are still there, but all the apps links in the menu are gone. How can I restore them?

**Comment by HUMORCE** [comment, marked as answer]
`scoop reset *`

**Reply by Miqueas** [reply]
That doesn't uninstall any apps?

**Reply by Miqueas** [reply]
Nvm, it works, thanks

**Reply by HUMORCE** [reply]
> That doesn't uninstall any apps?

No, it will recreate shims, shortcuts, environment variables, etc. for installed apps.

**Reply by Miqueas** [reply]
> > That doesn't uninstall any apps?
> 
> No, it will recreate shims, shortcuts, environment variables, etc. for installed apps.

Yeah, I executed it and that's what happened, thank you
